spec/features/observer_spec.rb

```diff
@@ -39,7 +39,7 @@
   it "allows a user to add a reason when adding an observer" do
     reason = "is the archbishop of banterbury"
     proposal = create(:proposal)
-    observer = create(:user)
+    observer = create(:user, client_slug: nil)
```

Comment:
ah!! let's get rid of that (not necessarily now)! Should be set explicitly, whether in specs or through trait etc